zenodo/modules/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -463,15 +489,24 @@
         }
       }
     },
-    "thesis_university": {
-      "description": "Awarding university in case of a thesis.",
-      "type": "string"
-    },
-    "thesis_supervisors": {
-      "description": "Supervisors of thesis in order of importance.",
-      "type": "array",
-      "items": {
-        "$ref": "#/definitions/person_or_org"
+    "thesis": {
```

Comment:
The ``thesis_university`` and ``thesis_supervisors`` merged under ``thesis`` is the only important change. Is it OK?